static/js/components/Candidate.jsx

```diff
@@ -45,75 +69,111 @@ const Candidate = ({ route }) => {
   }
 
   return (
-    <div className={styles.source}>
-      <div className={styles.leftColumn}>
-        <div className={styles.name}>{candidate.id}</div>
-        <br />
-        <b>Position (J2000):</b>
-        &nbsp;
-        {candidate.ra}, &nbsp;
-        {candidate.dec}
-        &nbsp; (&alpha;,&delta;=
-        {ra_to_hours(candidate.ra)}, &nbsp;
-        {dec_to_hours(candidate.dec)}) &nbsp; (l,b=
-        {candidate.gal_lon.toFixed(6)}, &nbsp;
-        {candidate.gal_lat.toFixed(6)}
-        )
-        <br />
-        <b>Redshift: &nbsp;</b>
-        {candidate.redshift}
-        <ThumbnailList
-          ra={candidate.ra}
-          dec={candidate.dec}
-          thumbnails={candidate.thumbnails}
-        />
-        <br />
-        <br />
-        <Responsive
-          element={FoldBox}
-          title="Photometry"
-          mobileProps={{ folded: true }}
-        >
-          <Plot
-            className={styles.plot}
-            url={`/api/internal/plot/photometry/${candidate.id}`}
-          />
-        </Responsive>
-        <Responsive
-          element={FoldBox}
-          title="Spectroscopy"
-          mobileProps={{ folded: true }}
-        >
-          <Plot
-            className={styles.plot}
-            url={`/api/internal/plot/spectroscopy/${candidate.id}`}
-          />
-        </Responsive>
-        {/* TODO 1) check for dead links; 2) simplify link formatting if possible */}
-        <Responsive
-          element={FoldBox}
-          title="Surveys"
-          mobileProps={{ folded: true }}
-        >
-          <SurveyLinkList
-            id={candidate.id}
-            ra={candidate.ra}
-            dec={candidate.dec}
-          />
-        </Responsive>
-      </div>
-
-      <div className={styles.rightColumn}>
-        <Responsive
-          element={FoldBox}
-          title="Comments"
-          mobileProps={{ folded: true }}
-          className={styles.comments}
-        >
-          <CommentList isCandidate />
-        </Responsive>
+    <Paper elevation={1}>
+      <div className={classes.source}>
+        <div className={classes.mainColumn}>
+          <div className={candidateStyles.topRow}>
+            <div className={classes.column}>
+              <div>
+                <div className={classes.alignRight}>
+                  <SharePage />
+                </div>
+                <div className={classes.name}>{candidate.id}</div>
+              </div>
+              <div>
+                <b>Position (J2000):</b>
+                &nbsp;
+                {candidate.ra}, &nbsp;
+                {candidate.dec}
+                &nbsp; (&alpha;,&delta;=
+                {ra_to_hours(candidate.ra)}, &nbsp;
+                {dec_to_hours(candidate.dec)}) &nbsp; (l,b=
+                {candidate.gal_lon.toFixed(1)}, &nbsp;
+                {candidate.gal_lat.toFixed(1)}
```

Comment:
I think this is `toFixed(5)` in Sources. Right @guynir42 @dannygoldstein?

Reply:
toFixed(6)